frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -28,10 +28,34 @@ import Head from "next/head";
 import CellGuideBottomBanner from "../CellGuideBottomBanner";
 import { useCellTypeMetadata } from "src/common/queries/cellGuide";
 import {
+  CELL_GUIDE_CARD_GLOBAL_ORGANISM_FILTER_DROPDOWN,
+  CELL_GUIDE_CARD_GLOBAL_TISSUE_FILTER_DROPDOWN,
   CELL_GUIDE_CARD_HEADER_NAME,
   CELL_GUIDE_CARD_HEADER_TAG,
   CELL_GUIDE_CARD_SYNONYMS,
 } from "src/views/CellGuide/components/CellGuideCard/constants";
+import DropdownSelect from "./components/common/DropdownSelect";
+import { SelectChangeEvent } from "@mui/material/Select";
+import { useOrganAndOrganismFilterListForCelltype } from "./components/MarkerGeneTables/hooks/common";
+import {
+  ALL_TISSUES,
+  NO_ORGAN_ID,
+} from "./components/MarkerGeneTables/constants";
+
+function getDefaultOrganSelection(
```

Comment:
Does this return the first tissue that is found in  the input array of organs?

Shouldn't we always want the default tissue to be "All Tissues"?